### ISSUE TITLE
new PRs blocked on a smoke test regression

### ISSUE THREAD

**Issue by kellyselden** [ember-cli/ember-cli#4588]
My assumption is a semver violation in one of our deps.
```
  1) Acceptance: smoke-test ember new foo, build production and verify fingerprint:

     AssertionError: vendor-981cf3132948d7641af94c08b953774a.js contains the fingerprint (5078db5185d245671a52cb5c71cbee68): expected 'vendor-981cf3132948d7641af94c08b953774a.js' to include '5078db5185d245671a52cb5c71cbee68'
```
This can track discussion and a possible fix.

**Comment by stefanpenner** [comment]
seems related to broccli-asset-rev's recent bump?

cc @rickharrison

**Comment by rickharrison** [comment]
We fixed a bug with how the md5 hash was generated. So the failing tests might just need to be modified for the fixed hashes. Behavior should still be working as intended

**Comment by stefanpenner** [comment]
@rickharrison sounds good.

@kellyselden mind updating the test in question?

@rickharrison can you direct me to the commit in question, I just want to be sure I understand the context correctly.

**Comment by rickharrison** [comment]
I'm pulling down master right now and will take a look at fixing the tests.

**Comment by rickharrison** [comment]
https://github.com/rickharrison/broccoli-asset-rev/pull/67

**Comment by kellyselden** [comment]
If this isn't resolved by tonight, I can look into it.

**Comment by kellyselden** [comment]
Fixed on #4589. Thanks everyone for speedy action.
